test/functional/apps/console/monaco/_autocomplete.ts

```diff
@@ -261,21 +261,6 @@ GET _search
         );
         expect(await PageObjects.console.monaco.isAutocompleteVisible()).to.be.eql(true);
       });
-
-      // not fixed for monaco yet https://github.com/elastic/kibana/issues/184442
-      it.skip('should not activate auto-complete after comma following endpoint in URL', async () => {
-        await PageObjects.console.monaco.enterText('GET _search');
-
-        await PageObjects.console.sleepForDebouncePeriod();
-        log.debug('Key type ","');
-        await PageObjects.console.monaco.enterText(','); // i.e. 'GET _search,'
-
-        await PageObjects.console.sleepForDebouncePeriod();
-        log.debug('Key type Ctrl+SPACE');
-        await PageObjects.console.monaco.pressCtrlSpace();
-
-        expect(await PageObjects.console.monaco.isAutocompleteVisible()).to.be.eql(false);
-      });
```

Comment:
I decided to delete this test since I believe this restriction is not worth the effort to implement it: we would need to check if the last url token before the comma is a known endpoint and if yes, then not suggest anything. I believe it's okay to trigger suggestions for indices in this case even though the url won't be correct.